pylint/checkers/base.py

```diff
@@ -2044,7 +2029,7 @@ def _raise_name_warning(
         )
 
         self.add_message(warning, node=node, args=args, confidence=confidence)
-        self.stats["badname_" + node_type] += 1  # type: ignore
+        self.linter.stats.set_bad_name(node_type, 1)
```

Comment:
Are we setting the value of bad_names or are we instead increasing it by one ?

Reply:
I'll rename these methods!